src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/strategy/translateddocuments/UploadTranslatedDocumentDefaultStrategy.java

```diff
@@ -31,30 +29,21 @@ public CallbackResponse uploadDocument(CallbackParams callbackParams) {
         List<Element<CaseDocument>> updatedDocumentList = updateSystemGeneratedDocumentsWithTranslationDocument(
             callbackParams);
         CaseData updatedCaseData = callbackParams.getCaseData().toBuilder().systemGeneratedCaseDocuments(
-            updatedDocumentList)
-            .respondent1ClaimResponseDocumentSpec(getTranslatedDocumentAsCaseDocument(callbackParams))
+                updatedDocumentList)
+            .caseDataLiP(CaseDataLiP
+                             .builder()
```

Comment:
put .builder() in the same line with the Class it refers too

Reply:
fixed